### PR TITLE
New package: Samsung.EasyButtonManager version 1.0.0.21

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/111972&drop=dogfoodAlpha

### CODE REVIEW

**Comment by stephengillie** [comment]
image

File: manifests/s/Samsung/EasyButtonManager/1.0.0.21/Samsung.EasyButtonManager.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.EasyButtonManager
+PackageVersion: 1.0.0.21
+Installers:
+- InstallerUrl: https://org.downloadcenter.samsung.com/downloadfile/ContentsFile.aspx?CDSite=US&CttFileID=1908715&CDCttType=SW&ModelType=C&ModelName=NP-Q1U/P01/SEA&VPath=SW/200804/20080422154518125_Easy_Button_Manager_1.0.0.21.zip
```
Comment (Trenly) [inline manifests/s/Samsung/EasyButtonManager/1.0.0.21/Samsung.EasyButtonManager.installer.yaml:7]:
```suggestion
- InstallerUrl: https://downloadcenter.samsung.com/content/SW/200804/20080422154518125_Easy_Button_Manager_1.0.0.21.zip
```

File: manifests/s/Samsung/EasyButtonManager/1.0.0.21/Samsung.EasyButtonManager.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Samsung.EasyButtonManager
+PackageVersion: 1.0.0.21
+PackageLocale: en-US
+Publisher: Samsung
+PackageName: EASY BUTTON MANAGER
```
Comment (Trenly) [inline manifests/s/Samsung/EasyButtonManager/1.0.0.21/Samsung.EasyButtonManager.locale.en-US.yaml:8]:
```suggestion
PackageName: Easy Button Manager
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by stephengillie** [comment]
image

**Comment by stephengillie** [comment]
image

**Comment by upintheairsheep** [comment]
https://answers.microsoft.com/en-us/windows/forum/all/error-1275-this-driver-has-been-blocked-from/71e7cda2-7fa1-4d76-9836-8acc227b1a2a

**Comment by stephengillie** [comment]
> https://answers.microsoft.com/en-us/windows/forum/all/error-1275-this-driver-has-been-blocked-from/71e7cda2-7fa1-4d76-9836-8acc227b1a2a

Thanks for the suggestions. Unfortunately, these steps are beyond the current capabilities of the WinGet package manager. Feel free to open an Issue suggesting that they be added.

**Pull request closed without merging**